### PR TITLE
Add getPoint apis from registration class

### PR DESCRIPTION
**Pull request by nikhilkalige**
I have added 2  apis from the `registration` class : ` getPointXYZRGB` and ` getPointXYZ`. 

I do not have much experience with cython, so I have not completed the `getPointXYZRGB` api as it has some casting, but I have tested the `getPointXYZ` api and it works properly.

### CODE REVIEW

File: pylibfreenect2/libfreenect2.pyx [hunk 1]
```diff
@@ -781,6 +781,60 @@ cdef class Registration:
...
+        Returns
+        -------
+        tuple : (X coordinate of the 3-D point (meter),
+                 Y coordinate of the 3-D point (meter),
+                 Z coordinate of the 3-D point (meter),
+                 rgb)
+
+        """
+        cdef float x, y, z, rgb
+        self.ptr.getPointXYZRGB(undistored.ptr, r, c, x, y, z, rgb)
```
Comment (r9y9) [inline pylibfreenect2/libfreenect2.pyx]: You seem to forget to pass registered frame as function arg? i.e.:
```py
 self.ptr.getPointXYZRGB(undistored.ptr, registered.ptr, r, c, x, y, z, rgb)
```

**Comment by r9y9** [comment]
Thanks for your contribution! The changes look good to me except for my above comment.

> I have not completed the getPointXYZRGB api as it has some casting

I'm not sure what the casting problem is, but I think `getPointXYZRGB` should work fine as `getPointXYZ` does.

**Comment by nikhilkalige** [comment]
I fixed that error, but I meant the casting problem because of these lines in the library docs of [registration class](https://openkinect.github.io/libfreenect2/classlibfreenect2_1_1Registration.html)
```
rgb	Color of the 3-D point (BGRX). To unpack the data, use

const uint8_t *p = reinterpret_cast<uint8_t*>(&rgb);
uint8_t b = p[0];
uint8_t g = p[1];
uint8_t r = p[2];
```

**Comment by r9y9** [comment]
I see. I guess it's not too hard to implement in cython.

**Comment by r9y9** [comment]
Probably the following works:

```cython
cimport cython
from libcpp.cast cimport reinterpret_cast
from libc.stdint cimport uint8_t

# Workaround for use of pointer type in reinterpret_cast
# https://groups.google.com/forum/#!msg/cython-users/FgEf7Vrx4AM/dm7WY_bMCAAJ
ctypedef uint8_t* uint8_pt

cdef unsafe_reinterpret_float(float x, int idx):
    cdef uint8_t* ptr = reinterpret_cast[uint8_pt](&x);
    cdef uint8_t r = ptr[idx];
    return r
```

```cython
b = unsafe_reinterpret_float(rgb, 0)
g = unsafe_reinterpret_float(rgb, 1)
r = unsafe_reinterpret_float(rgb, 2)
```

**Pull request closed without merging**